data/example/README.md

```diff
@@ -0,0 +1,16 @@
+# Example TIDES Data Package
+
+Template directory for example scaffolding and helper scripts.
+
+## Scripts for Generating Data
+
+`scripts\create_example.py` has some template code which can help with the following
+
+- `write_schema_examples()`: will generate blank csvs according to the TIDES schema
+- `write_datapackage()` will generate a datapackage.json based on the TIDES schemas and a set of defaults specified in th script.
```

Comment:
I think I'd prefer a `datapackage.json` template instead of a function. Honestly, I'm still trying to figure out what the value of an empty generated example is that wouldn't be better served by a real sample?

Reply:
I don't think the functions serve much purpose other than for developers to be able to auto-generate the example `datapackage.json` and CSV templates.